### PR TITLE
remove the `exception`

### PR DESCRIPTION
**Pull request by HandyKurniawan**

### CODE REVIEW

File: quconot/implementations/mct_no_auxiliary_relative.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def verify_mct_cases(
         relative_phase: true / false (D)
         clean_auxiliary: true (D) / false
         wasted_auxiliary: true / false (D)
-        separable_wasted_auxiliary: true / false (D)    `requires wasted_auxiliary set to True`
```
Comment (adamglos92) [inline quconot/implementations/mct_no_auxiliary_relative.py:40]: this is strange, what happened here?

File: test/implementations/functions_testing.py [hunk 2]
```diff
@@ -576,90 +557,88 @@ def verify_circuit_dirty_wasted_relative_separable_auxiliary(
...
+    ):
+        return False, "Something wrong with the implementation"
 
-        return True, ""
-    except Exception:
-        return False, ""
+    return True, ""
 
 
-def verify_dependencies(rd):
```
Comment (adamglos92) [inline test/implementations/functions_testing.py:634]: why it is commented out? I'm not saying it is bad, but I want to understand

Comment (HandyKurniawan) [reply test/implementations/functions_testing.py:634]: i put it in each of the implementations, test_dependencies.

Also, I will fix this with another issue with the recursivel.. that is why I commented them out.

File: quconot/implementations/mct_no_auxiliary_relative.py (hunk 1 above)
Comment (HandyKurniawan) [reply quconot/implementations/mct_no_auxiliary_relative.py:40]: last time you asked to remove the `#`, so I put the ` apostrophe ` to make them readable in the documentation

File: test/implementations/functions_testing.py (hunk 2 above)
Comment (HandyKurniawan) [reply test/implementations/functions_testing.py:634]: will be fixed in issue #15

File: quconot/implementations/mct_no_auxiliary_relative.py (hunk 1 above)
Comment (HandyKurniawan) [reply quconot/implementations/mct_no_auxiliary_relative.py:40]: will be fixed in issue #13

**Pull request merged**